src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/AdditionalServices.feature

```diff
@@ -131,3 +131,86 @@ Scenario: Additional Services - Select service recipient - Go back
+	When they choose to continue
+	And the User chooses to save
+	Then the Additional Service is not saved
+	And the reason is displayed
+
+Scenario: Additional Services - Data exceeds the maximum length
+	Given the Order Form for the existing order is presented
+	And the available prices for the selected Additional Service are presented
+	And the User has selected a Additional Service price
+	And the User has selected a Service Recipient
+	When they choose to continue
+	And the quantity is above the max value
+	And the User chooses to save
+	Then the Additional Service is not saved 
+	And the reason is displayed
+
+Scenario: Additional Services - Validation Error Message Anchors
+	Given the Order Form for the existing order is presented
+	And the available prices for the selected Additional Service are presented
+	And the User has selected a Additional Service price
+	And the User has selected a Service Recipient
+	When they choose to continue
+	And the User chooses to save
+	And the user selects an error link in the Error Summary
+	Then they will be navigated to the relevant part of the page
+
+Scenario: Additional Services - All data are valid
+	Given the Order Form for the existing order is presented
+	And the available prices for the selected Additional Service are presented
+	And the User has selected a Additional Service price
+	And the User has selected a Service Recipient
+	When they choose to continue
+	And all data is complete and valid
+	And the User chooses to save	
+	Then the Additional Service is saved
+	And the section content validation status of the Additional Service is Complete
+	And the delete button is enabled
+
+Scenario: Additional Services - Go back before save
+	Given the Order Form for the existing order is presented
+	And the available prices for the selected Additional Service are presented
+	And the User has selected a Additional Service price
+	And the User has selected a Service Recipient
+	When they choose to continue
+	And the User chooses to go back
+	Then they are presented with the Service Recipients saved in the Order
+
+Scenario: Additional Services - Go back post save
+	Given the edit Additional Service form for flat list price with variable (patient numbers) order type is presented
+	And the Order Form for the existing order is presented
+	And the User is able to manage the Additional Services section
+	And the User chooses to edit the saved Additional service
+	When the User chooses to go back
+	Then the Additional Service dashboard is presented
+
+Scenario: Additional Services - Values populated after editing and saving
+	Given the edit Additional Service form for flat list price with variable (patient numbers) order type is presented
+	And the Order Form for the existing order is presented
+	And the User is able to manage the Additional Services section
+	When the User chooses to edit the saved Additional service	
+	Then the Quantity is populated
+	And the Price is populated
```

Comment:
I'd replace the last two steps with this step
`the values will be populated with the values that was saved by the User`
It makes use of some variables saved in the Context during the initial save so you testing that the value you entered is the value that's populated